sound/soc/sof/topology.c

```diff
@@ -3216,7 +3232,7 @@ static int sof_link_alh_load(struct snd_soc_component *scomp, int index,
 	config->hdr.size = size;
 
 	/* set config for all DAI's with name matching the link name */
-	ret = sof_set_dai_config(sdev, size, link, config);
+	ret = sof_set_dai_config(sdev, size, link, config, 1, 0);
```

Comment:
Lists of many literal numbers in function calls are always a bit hard to read. I wonder would this be easier to read if you'd add sof_set_dai_config_multi() that takes the two extra arguments, and define 

sof_set_dai_config(x,y,z) { return sof_set_dai_config_multi(x, y, z, 1, 0); } 

... so non-SSP code would stay the same.

Reply:
hmm, yes good idea, I'll try it out.